### DISCUSSION TITLE
Non-functioning code in official documentation

### DISCUSSION THREAD

**Discussion by VOJAKIS** [primefaces/community#2665] [PrimeReact] [answered]
Hi, the code for Multiselect with the FloatLabel component is not working correctly. The FloatLabel does not have any 'className' props.

FloatLabel documentation: https://primereact.org/floatlabel/
MultiSelect documentation: https://primereact.org/multiselect/#floatlabel

![image](https://github.com/user-attachments/assets/70732ad0-2ba7-4764-9aa8-ec05c55e8b42)

**Comment by melloware** [comment, marked as answer]
I opened a ticket: https://github.com/primefaces/primereact/issues/7182

**Reply by melloware** [reply]
10.8.3 is released and this looks fixed to me.
